subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -167,6 +180,8 @@ isoal_status_t sink_sdu_emit_hci(const struct isoal_sink             *sink_ctx,
 				 const struct isoal_emitted_sdu_frag *sdu_frag,
 				 const struct isoal_emitted_sdu      *sdu)
 {
+	const struct device *dev = DEVICE_DT_GET(DT_DRV_INST(0));
```

Comment:
This is not particularly clean, however I didn't immediately see a way to get the `const struct device *` routed to the `sink_sdu_emit_hci()` call. In all the other cases (e.g. the rx threads) I was able to pass the `struct device` as a callback parameter.